index.html

```diff
@@ -22,7 +22,6 @@ <h1 id="title">Thomas Edison The Genius</h1>
       </figure>
       <section id="tribute-info">
         <div id="text-containers">
```

Comment:
```suggestion
      <section id="tribute-info">
        <div id="text-containers">
```
- It’s better to apply classes to control the style of multiple of the same type of element and more abstracted logic than to create many #id’s.

Reply:
- [ ] I think it would be better if you explain why you request a change - it might be obvious for you but not for someone else. For example, you can say **"Why don't you use _class_ instead of _id_ to apply a style for multiple elements, that way you can avoid repeating the same code for each HTML element that uses the same styling"**